pkg/utils/utils.go

```diff
@@ -234,3 +234,46 @@ func SetAvailableResourcesForApi(groupVersion string, resources *metav1.APIResou
 
 	gvResourcesCache[groupVersion] = resources
 }
+
+func GetEnvVarValue(envVars []v1.EnvVar, key string) (string, bool) {
+	for _, envVar := range envVars {
+		if envVar.Name == key {
+			return envVar.Value, true // if key exist, return value, true
+		}
+	}
+	return "", false // if key does not exist, return "", false
+}
+
+// IsUnknownGpuResourceType check if the provided gpu resource is unknown one
+func IsUnknownGpuResourceType(resources v1.ResourceRequirements) bool {
```

Comment:
what happens if 2 GPUS are used? Is it supported or not atm?

Reply:
I’m not sure how common it is to use two different GPUs. Therefore, I think it would be appropriate to consider supporting such requests if they arise. Currently, we are assuming that only one GPU type will be used.